.github/scripts/oss-build.functions.sh

```diff
@@ -73,3 +73,23 @@ function should_build_oss() {
 
   echo "no"
 }
+
+function get_hz_dist_zip() {
+  local hz_variant=$1
+  local hz_version=$2
+  local suffix=$3
+
+  # The slim is an artifact with a classifier, need to add `-` there
+  if [[ -n "${hz_variant}" ]]; then suffix="-${hz_variant}"; fi
+
+  if [[ "${hz_version}" == *"SNAPSHOT"* ]]
+  then
+      # DI-95 - Do not rely on the OSS distribution zip in the hazelcast-docker PR builder
+      # https://hazelcast.atlassian.net/browse/DI-95
+      url="$(aws s3 presign s3://hazelcast/distribution-snapshot/hazelcast-"${hz_version}""${suffix}".zip --expires-in 600)"
```

Comment:
What about other non-snapshot versions? e.g. 5.4.1 or 5.4.1-DEVEL/BETA-1?

FYI @nishaatr

Reply:
Perhaps BETA as DEVEL is made via another workflow

Reply:
Good point - _going forwards_, will BETA releases be public?

Reply:
On second thought, lets leave BETA as planning to cleanup in the future when actual release is made
I think future BETA will be EE only unless we are told otherwise